<launch>
   <!-- Start the Dynamixel Joint States Publisher -->

  <include file="$(find pr2lite_moveit_config)/launch/id_server.launch" />
  <include file="$(find pr2lite_moveit_config)/launch/pc_485net_with_delay.launch" />

<!--
  <node pkg="pr2lite_moveit_config" name="dynamixel_joint_states_publisher" type="dynamixel_joint_state_publisher.py" output="screen">
      <param name="rate" value="5" />
      <rosparam file="$(find pr2lite_moveit_config)/config/pr2lite_arm.yaml" command="load" />
  </node>
  <include file="$(find pr2lite_moveit_config)/launch/festival.launch" />
  <include file="$(find pr2lite_moveit_config)/launch/pr2lite_vision.launch" />
-->

  <param name="/use_sim_time" value="false"/>
  <!-- send pr2lite urdf to param server -->
  <param name="robot_description" command="cat '$(find pr2lite_moveit_config)/config/pr2lite_urdf.xml'" />

  <node name="statepub" pkg="robot_state_publisher" type="robot_state_publisher" />

  <node pkg="diagnostic_aggregator" type="aggregator_node" name="diagnostic_aggregator" >
      <rosparam command="load" file="$(find pr2lite_moveit_config)/config/pr2lite_diagnostics.yaml" />
  </node>
  <include file="$(find pr2lite_actuators)/launch/linact_server.launch" />
  <node name="rob_st_pub" pkg="robot_state_publisher" type="robot_state_publisher" >
      <param name="publish_frequency" value="5.0" />
  </node>

  <node pkg="pr2lite_moveit_config" type="joint_states_listener.py" name="joint_states_listener" />

  <include file="$(find pr2lite_nav)/launch/pr2lite_bringup_base.launch" />
  <include file="$(find pr2lite_moveit_config)/launch/fake_pub.launch" />

  <include file="$(find pr2lite_moveit_config)/launch/pr2lite_nav_joy.launch" />
<!--
-->
</launch>
